master/pkg/searcher/pbt.go

```diff
@@ -208,7 +208,9 @@ func (s *pbtSearch) progress(unitsCompleted float64) float64 {
 		s.LengthPerRound.MultInt(s.PopulationSize).MultInt(s.NumRounds).Units)
 }
 
-func (s *pbtSearch) trialExitedEarly(ctx context, requestID RequestID) ([]Operation, error) {
+func (s *pbtSearch) trialExitedEarly(
+	ctx context, requestID RequestID, exitedReason workload.ExitedReason,
```

Comment:
```suggestion
	ctx context, requestID RequestID, _ workload.ExitedReason,
```
similar comment elsewhere.

Reply:
Eventually I'm going to be using the `exitedReason` here once I add in searcher-specific functionality, so thinking I should keep it like this for now rather than change it to `_`?